docs/guides/proxies/haproxy_apache_lxd.md

```diff
@@ -8,7 +8,7 @@ tested_with: 8.5, 8.6, 9.0
 
 ## Introduction
 
-HAProxy stands for "High Availability Proxy." This proxy can sit in front of any TCP application (such as web servers), but it is often used to act as a load-balancer between multiple instances of a website.
+HAProxy stands for "High Availability Proxy". This proxy can sit in front of any TCP application (such as web servers), but it is often used to act as a load-balancer between multiple instances of a website.
```

Comment:
I actually think that the period should be inside the quotes here. It's not a code block, which would be the only exception I can think of. See this: https://docs.rockylinux.org/guides/contribute/style_guide/#grammar-and-punctuation and the bullet on periods and commas.